clang/test/CheckedCRewriter/b9_allsafestructp.c

```diff
@@ -17,29 +22,35 @@ struct p {
   int *x;
   char *y;
 };
-//CHECK: _Ptr<int> x;
```

Comment:
Why have these gone away? Seems like all of the `CHECK` annotations you had before should be `CHECK_NOALL` now, right?

Reply:
Ah, I added annotations almost purely to the functions as opposed to fields of structs I used. Thanks for pointing this one out. I can change the tests to reflect this!